src/Server.h

```diff
@@ -241,6 +245,8 @@ class cServer
 	Initialized in InitServer(), used in Start(). */
 	AStringVector m_Ports;
 
+	cChannelManager m_ChannelManager;
+
```

Comment:
I think it might be better to make it a `UniquePtr<cChannelManager>`, that way you don't need to `#include` the class declaration file here (include it in the cpp instead) and thus it reduces compile-time dependencies.